### DISCUSSION TITLE
New files have been added

### DISCUSSION THREAD

**Discussion by GeorgeGeorgeGe** [rejetto/hfs#336] [Ideas]
Hello,

Is there anyway to know the most recent files has been uploaded to the system? some users added files/folders deep in the folder structure, other "target" users have to "guess" where and when the files added"

Regards

**Comment by rejetto** [comment]
there's nothing like that at the moment, but it's surely an interesting feature.
A plugin could do it.
I have to think if to have it as plugin or not.

**Reply by GeorgeGeorgeGe** [reply]
Thanks a lot.

**Reply by GeorgeGeorgeGe** [reply]
do we expect to see the recent activities list soon?
Thanks.

**Comment by rejetto** [comment]
i can hope before the end of the year, but all depends on my free time

**Reply by GeorgeGeorgeGe** [reply]
Ok, Thanks

**Comment by GeorgeGeorgeGe** [comment]
Hello,
any update about this possible feature ?
Thanks

**Comment by rejetto** [comment]
not yet, it will be probably part of 0.53
i've been delaying this because it needs storing data, and i'm struggling at deciding how.
i will keep you posted here.

**Comment by GeorgeGeorgeGe** [comment]
Thanks a lot.


Sent from Yahoo Mail for iPhone


On Monday, February 19, 2024, 4:05 AM, Massimo Melina ***@***.***> wrote:




not yet, it will be probably part of 0.53
i've been delaying this because it needs storing data, and i'm struggling at deciding how

**Comment by rejetto** [comment]
hey, i was reviewing this request, and i noticed i got it wrong from the beginning.
I confused it with another feature of HFS 2, sorry.
Re-reading your first message, you need a log of recent uploads, and you already have that!
Go to Admin-panel > Logs
go with your mouse on "Notes"
click Filter
enter "upload"

**Reply by hvscan** [reply]
I hate to comment irrelevantly but only contact I could find for you without having an account,  could you please fix the email sign up, I have been waiting for an email confirmation so I can comment on the forums regarding HFS chat. Have requested multiple times and checked spam

Sorry and thank you

**Reply by rejetto** [reply]
hey, sorry to hear that.
you can contact me at a@rejetto.com
i cannot find your account (hvscan?) among the hundreds of spam accounts.

**Reply by hvscan** [reply]
Appreciate the prompt reply very much, will shoot you an email instead of clogging up this discussion, feel free to purge all of this. Thank you.

**Reply by rejetto** [reply]
#583

**Comment by GeorgeGeorgeGe** [comment]
> hey, i was reviewing this request, and i noticed i got it wrong from the beginning. I confused it with another feature of HFS 2, sorry. Re-reading your first message, you need a log of recent uploads, and you already have that! Go to Admin-panel > Logs go with your mouse on "Notes" click Filter enter "upload"

Hi rejetto,
i did not want that feature for the admin, i needed for the users. it is like a notification that a user has recently added or uploaded a file and its path.
Thanks.

**Comment by rejetto** [comment]
i see.
i already planned to soon add a feature where new files are "flagged", as in HFS 2, but that's not a dedicated list, it's just a marker appearing next to the file name, so you still have to go "deep in the folder structure" as you said.
you instead wanted a list, right?
while what you mean by "uploaded" is clear, "added" is not.

**Comment by GeorgeGeorgeGe** [comment]
Flagged as "new" could do the job as well. 
I hope we see that feature soon.
Thanks a lot.

**Comment by rejetto** [comment]
have a look at this https://github.com/rejetto/hfs/discussions/584
